model/bin/make_makefile.sh

```diff
@@ -922,7 +922,7 @@
                prop=
              source="$pdlibcode $pdlibyow $db $bt $setupcode w3triamd $stx $nlx $btx $is w3parall $uostmd"
                  IO="w3iogrmd $oasismd $agcmmd $ogcmmd $igcmmd"
-                aux="constants w3servmd w3timemd w3arrymd w3dispmd w3gsrumd $tidecode w3nmlprncmd" ;;
+                aux="constants w3servmd w3timemd w3arrymd w3dispmd w3gsrumd w3tidemd w3nmlprncmd" ;;
```

Comment:
why was this changed from $tidecode to w3tidemd?  None of the other $tidecode options were changed?

Reply:
it's because the tide feature is now implemented in ww3_prnc with MPI implementation so it was hard to handle both !/MPI switch and !/TIDE switch in the code for the same lines. By the way, there is a flag TIDEFLAG to only compute TIDE feature if requested.